### PR TITLE
Add a GitHub 'fork me' ribbon or a badge

### LINKED ISSUE #38: Add a GitHub "fork me" ribbon or a badge

### PR DESCRIPTION
**Pull request by andersonleite**
Hey @therealklanni 
I added the ribbon to the main page.
Also changed css a bit so it doesn't stay on top of the logo.

Closes #38

### CODE REVIEW

**Comment by therealklanni** [comment]
@andersonleite thanks for the PR! Looks like we need to style it properly for mobile, however. I'll let @troym9731 give you some feedback there, since he did the site design.

**Comment by troym9731** [comment]
Awesome! Thanks for contributing! However, I'm not sure this ribbon is the right fit for the site, considering we have to move around the nav just to compensate. This all CSS implementation https://simonwhitaker.github.io/github-fork-ribbon-css/ seems like it would allow us to put it in the bottom right, not fixed, and easily make it responsive without having to move things around.

**Comment by therealklanni** [comment]
I like that approach. @andersonleite want to take a shot at it? :smile:

**Comment by andersonleite** [comment]
@therealklanni @troym9731 
sure, can you check it?

File: views/layouts/main.hbs [hunk 1]
```diff
@@ -6,6 +6,10 @@
   <meta name="description" content="Slack Invite Request">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="stylesheet" type="text/css" href="//maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/github-fork-ribbon-css/0.2.0/gh-fork-ribbon.min.css" />
+  <!--[if lt IE 9]>
```
Comment (therealklanni) [inline views/layouts/main.hbs]: We don't support IE :grinning: 

You can take this out.

**Comment by therealklanni** [comment]
Please remove `npm-debug.log`

**Comment by therealklanni** [comment]
@andersonleite something doesn't look right.

Imgur

**Comment by andersonleite** [comment]
hey @therealklanni 
try it again, please.

**Comment by therealklanni** [comment]
Looks good. Can you just override 

``` css
.github-fork-ribbon {
  position: fixed;
}
```

?

**Comment by andersonleite** [comment]
@therealklanni 
Like this?

**Comment by troym9731** [comment]
Sorry to make you change it again, but instead of having the `.github-fork-ribbon` be `position: fixed;`, instead add `position: relative` to the `.page` class, and then that fixes some wonkiness of the ribbon not being at the bottom of the page. After that, it all looks great!

**Comment by andersonleite** [comment]
@troym9731 
no worries man, just guide me here until we have it perfect.
can you take a look now?

**Comment by troym9731** [comment]
Looks great! I'll go ahead and merge it.

**Comment by therealklanni** [comment]
@andersonleite sorry to mislead you. That's what I get for meddling with @troym9731's design!

**Pull request merged**